docker-compose.yml

```diff
@@ -1,5 +1,5 @@
 hawkular:
-  image: "hawkular/hawkular-services:latest"
+  image: "hawkular/hawkular-services:devel"
```

Comment:
Not sure of this:
Using devel means that we will testing against master hawkular-services, thus if there is problem (on services), it will make our build hard fail.
On the other hand it will fail earlier, as opposed to a services release, but Hawkular-services PRs already run a test round against master version of the gem (see [here](https://github.com/hawkular/hawkular-services/pull/171#issuecomment-319419439)).
I was also thinking that we could lock a hawkular-services version for test (i.e., version instead of latest), and bumping as we support newer versions.